### ISSUE TITLE
add python linter (e.g. from drake) to ctest 

### ISSUE THREAD

**Issue by RussTedrake** [RussTedrake/underactuated#77]
@jamiesnape -- would appreciate if you have any pointers on this?

**Comment by jamiesnape** [comment]
I haven't tested this, but something like the following should hopefully get you started:

```cmake
find_program(PYLINT_EXECUTABLE pylint)  # pip install pylint

if(PYLINT_EXECUTABLE)
  add_test(NAME pylint COMMAND find "${PROJECT_SOURCE_DIR}" -name *.py -exec "${PYLINT_EXECUTABLE}" {} \;)
endif()

**Comment by jamiesnape** [comment]
OpenCV has something a little more complicated: https://github.com/opencv/opencv/commit/d195f2775abdd7d5cb3bfba7f84f0631492e1062.

**Comment by RussTedrake** [comment]
awesome.  will try it tonight.  thanks!

**Comment by RussTedrake** [comment]
resolved by #78
